spec/requests/customer_resources_spec.rb

```diff
@@ -467,6 +497,38 @@
         end
       end
 
+      describe 'setting a coverage statement' do
+        let(:params) do
+          {
+            'data' => {
+              'type' => 'customerResources',
+              'attributes' => {
+                'isSelected' => true,
+                'coverageStatement' => 'Only 1990s issues available.'
+              }
+            }
+          }
+        end
+
+        before do
+          VCR.use_cassette('put-customer-resources-coveragestatement-update') do
+            put '/eholdings/customer-resources/22-1887786-1440285',
+                params: params, as: :json, headers: update_headers
+          end
+        end
+
+        it 'responds with OK status' do
+          expect(response).to have_http_status(200)
+        end
+
+        let!(:json) { Map JSON.parse response.body }
```

Comment:
this isn't directly related to this PR but i think we're using `let` way incorrectly throughout the suite.  i'm gonna make a note to go clean that up...